CETCompression.md

```diff
@@ -41,9 +41,8 @@ digits where signatures were ignored.
 ## Adaptor Points with Multiple Signatures
 
 Given public key `P` and nonces `R1, ..., Rn` we can compute `n` individual signature points for
-a given event `(d1, ..., dn)` in the usual way: `si * G = Ri + H(P, Ri, di)*P`.
-To compute a composite adaptor point for all events which agree on the first `m` digits, where
-`m` is any positive number less than or equal to `n`, the sum of the corresponding signature
+a given event `(d1, ..., dn)` as follow: `si * G = Ri + cj * P` where `cj` is the value of the digit being signed.
```

Comment:
also, just to clarify with @LLFourn, it is okay to use `d_i = 0` correct? Even though it reveals the nonce key?